### PR TITLE
Update openshift-console-dashboard.json

### PR DESCRIPTION
**Pull request by MazArslan**
### What does this PR do?

Updating The `openshift-console-dashboard` to include new metrics: 

- total workspace CPU usage
- average workspace CPU usage
- total workspace Memory usage
- average workspace Memory usage
- Number of Running Workspaces
- Node CPU Usage as Percentage
- Node Memory Utilisation as Percentage
- Node CPU Usage as graph
- Node Memory Utilisation as graph

### What issues does this PR fix or reference?

No issue, Just improvements.

### Is it tested? How?

Update `grafana-dashboard-dwo` Config Map in `openshift-config-managed` NameSpace in Openshift to new Json. 

### PR Checklist

- [ ] E2E tests pass (when PR is ready, comment `/test v8-devworkspace-operator-e2e, v8-che-happy-path` to trigger)
    - [ ] `v8-devworkspace-operator-e2e`: DevWorkspace e2e test
    - [ ] `v8-che-happy-path`: Happy path for verification integration with Che

### CODE REVIEW

**Review by amisevsk — APPROVED** [review]

**Review by dkwon17 — COMMENTED** [review]
Thank you @MazArslan, I just have a small suggestion and a question

File: docs/grafana/openshift-console-dashboard.json [hunk 1]
```diff
@@ -1,1017 +1,1475 @@
...
-      "links": [],
-      "mappingType": 1,
-      "mappingTypes": [
+            "datasource": null,
+            "format": "percentunit",
+            "id": 57,
+            "targets": [
+                {
+                    "exemplar": true,
+                    "expr": "   (sum( 1 - sum without (mode) (rate(node_cpu_seconds_total{mode=~\"idle|iowait|steal\"}[2m])) * on(instance) group_left() (label_replace(max by (node) (kube_node_role{role=~\".+\"}), \"instance\", \"$1\", \"node\",\"(.*)\")))/  count(node_cpu_info)) ",
```
[the remark this answers is not in the document]
Comment (MazArslan) [reply docs/grafana/openshift-console-dashboard.json:329]: You are correct, Fully not needed. Will Remove :)

File: docs/grafana/openshift-console-dashboard.json [hunk 2]
```diff
@@ -1,1017 +1,1475 @@
...
+                {
+                    "exemplar": true,
+                    "expr": "sum(rate(container_cpu_usage_seconds_total{container=\"\"}[1m]) * on(pod) group_left kube_pod_labels{label_controller_devfile_io_creator!=''}) / count(kube_pod_info{pod=~\"^workspace.*\"})",
+                    "interval": "",
+                    "legendFormat": "AVG CPU Usage",
+                    "queryType": "randomWalk",
+                    "refId": "A"
+                }
+            ],
+            "title": "Average CPU Workspace Usage",
```
[the remark this answers is not in the document]
Comment (MazArslan) [reply docs/grafana/openshift-console-dashboard.json]: Done :+1:

File: docs/grafana/openshift-console-dashboard.json [hunk 3]
```diff
@@ -1,1017 +1,1475 @@
...
+                "min": false,
+                "show": true,
+                "total": false,
+                "values": false
+            },
+            "span": 6,
+            "targets": [
+                {
+                    "exemplar": true,
+                    "expr": " sum( 1 - sum without (mode) (rate(node_cpu_seconds_total{mode=~\"idle|iowait|steal\"}[2m])) * on(instance) group_left() ( label_replace(max by (node) (kube_node_role{role=~\".+\"}), \"instance\", \"$1\", \"node\",\"(.*)\") ) )",
```
Comment (dkwon17) [inline docs/grafana/openshift-console-dashboard.json:378]: @MazArslan great thank you, I'm wondering if 
```
* on(instance) group_left() ( label_replace(max by (node) (kube_node_role{role=~\".+\"}), \"instance\", \"$1\", \"node\",\"(.*)\")
```
can also be removed here?

**Review by dkwon17 — APPROVED** [review]

**Pull request merged**